src/csharp/Microsoft.Spark/Sql/DataFrameReader.cs

```diff
@@ -114,8 +114,20 @@ public DataFrameReader Options(Dictionary<string, string> options)
         /// </remarks>
         /// <param name="paths">Input paths</param>
         /// <returns>DataFrame object</returns>
-        public DataFrame Load(params string[] paths) =>
-            new DataFrame((JvmObjectReference)_jvmObject.Invoke("load", (object)paths));
+        public DataFrame Load(params string[] paths)
```

Comment:
Instead of doing this, let's expose, `Load()` and `Load(string path)`.

Reply:
Ok